Bigtable/src/BigtableClient.php

```diff
@@ -0,0 +1,202 @@
+<?php
+/*
+ * Copyright 2018, Google LLC All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Bigtable;
+
+use Google\Auth\FetchAuthTokenInterface;
+use Google\Cloud\Core\ClientTrait;
+use Google\Cloud\Core\Exception\GoogleException;
+use Google\Cloud\Core\Int64;
+use Google\Cloud\Core\LongRunning\LongRunningOperation;
+use Google\Cloud\Core\LongRunning\LROTrait;
+use Google\Cloud\Bigtable\Admin\V2\BigtableInstanceAdminClient as InstanceAdminClient;
+use Google\Cloud\Bigtable\Connection\Grpc;
+use Google\Cloud\Bigtable\Connection\LongRunningConnection;
+
+/**
+ * Cloud Bigtable is a highly scalable, transactional, managed, NewSQL
+ * database service. Find more information at
+ * [Cloud Bigtable docs](https://cloud.google.com/bigtable/).
+ *
+ * In production environments, it is highly recommended that you make use of the
+ * Protobuf PHP extension for improved performance. Protobuf can be installed
+ * via [PECL](https://pecl.php.net).
+ *
+ * ```
+ * $ pecl install protobuf
+ * ```
+ *
+ * Example
```

Comment:
I have a feeling it will be easy to forget writing snippet tests if we remove the standard for triggering their reminder. Could we add the colon back?